org.idempiere.zk.iceblue_c.theme/src/web/theme/iceblue_c/css/fragment/font-icons.css.dsp

```diff
@@ -358,6 +358,9 @@
 .z-icon-StepForward:before {
 	content: "\f04e";
 }
+.z-icon-Tab::before {
+	content: "\f24d";
+}
```

Comment:
Since we still include the breeze theme, please add this to the breeze theme too.
Also, icon image needs to be added for use case where user is using image instead of font icon.

Reply:
hi @uthadehikaru , still missing the image file. If you can't find a suitable one, you can create one from the font icon.